modules/core/src/v2/coins/btc.ts

```diff
@@ -85,17 +85,23 @@ export class Btc extends AbstractUtxoCoin {
     }).call(this);
   }
 
+  // using blockchair api: https://blockchair.com/api/docs#link_300
+  // https://api.blockchair.com/{:btc_chain}/dashboards/address/{:address}₀
+  // example utxo from response:
+  // {"block_id":-1,"transaction_hash":"cf5bcd42c688cb7c55b5811645e7f0d2a000a85564ca3d6b9fc20f57e14b30bb","index":1,"value":558},
   getUnspentInfoFromExplorer(addressBase58: string): Bluebird<any> {
     const self = this;
     return co(function *getUnspentInfoFromExplorer() {
-      const unspentInfo = yield request.get(self.recoveryBlockchainExplorerUrl(`/address/${addressBase58}/unspent`)).result();
+      const unspentInfo = yield request.get(self.recoveryBlockchainExplorerUrl(`/dashboards/address/${addressBase58}`)).result();
 
-      const unspents = unspentInfo.unspent;
+      const unspents = unspentInfo.data[addressBase58].utxo;
 
-      unspents.forEach(function processUnspent(unspent) {
-        unspent.amount = unspent.value_int;
+      unspents.forEach((unspent) => {
+        unspent.amount = unspent.value;
+        unspent.n = unspent.index;
+        unspent.txid = unspent.transaction_hash;
+        unspent.address = addressBase58;
```

Comment:
it would be a bit nicer if we used `.map()` so that we don't have duplicate attributes

also we should probably check the values (it's a shame that we don't have an unspent type)

Reply:
I changed the forEach to map. As for value checking, we have a very loose interface `UnspentInfo` right now, which only requires the property `address` to be present. I think in the future we should just tighten the interface of `UnspentInfo` in general, and we can add type checking for this tighter interface across the board.